scaffolding-chef-inspec/lib/scaffolding.sh

```diff
@@ -141,20 +141,12 @@ EOF
         "file": "{{pkg.svc_path}}/logs/inspec_last_run.json"
       }{{#if cfg.automate.enable ~}},
       "automate" : {
-        "url": "{{cfg.automate.server_url}}/data-collector/v0/",
```

Comment:
Can we leave off the `/data-collector/v0/` part here and have the `cfg.automate.server_url` have the full URL. This allows the user to change what API version they are using in Automate. Also, this makes it easier for testing, because one send to the report to any URL, for example, I have been testing with http://requestbin.sjcmmsn.com/.

Reply:
We'll have automated tests soon(!) I'm scheduled to work on them next week.

Ideally we don't expose the endpoint to the user at all. Currently this works slightly differently for Infra and InSpec, but the way InSpec is doing it is ideal.